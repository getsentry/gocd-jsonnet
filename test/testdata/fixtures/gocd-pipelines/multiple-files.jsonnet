local gocd_pipelines = import '../../../../libs/gocd-pipelines.libsonnet';

gocd_pipelines.pipelines_to_files_object([
  {
    name: 'example-1',
    pipeline: {
      group: 'example-1',
      materials: [],
    },
  },
  {
    name: 'example-2',
    pipeline: {
      group: 'example-2',
      materials: [],
    },
  },
])
