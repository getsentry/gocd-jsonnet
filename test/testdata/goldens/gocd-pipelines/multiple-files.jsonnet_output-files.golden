{
   "example-1.yaml": {
      "format_version": 10,
      "pipelines": {
         "example-1": {
            "group": "example-1",
            "materials": [ ]
         }
      }
   },
   "example-2.yaml": {
      "format_version": 10,
      "pipelines": {
         "example-2": {
            "group": "example-2",
            "materials": [ ]
         }
      }
   }
}
